templates/batch-llm/README.ipynb

```diff
@@ -5,9 +5,31 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Batch LLM using vLLM and Ray Data\n",
+    "## LLM offline batch inference with Ray Data and vLLM\n",
     "\n",
-    "This notebook will provide an example walking through how to use Ray Data and vLLM to run batch inference of a LLM."
+    "**⏱️ Time to complete**: 10 min\n",
+    "\n",
+    "This template shows you how to:\n",
+    "1. Read in data from in-memory samples or files on cloud storage. \n",
+    "2. Use Ray Data and vLLM to run batch inference of a LLM.\n",
+    "3. Write the inference outputs to cloud storage.\n",
+    "\n",
+    "For a Python script version of the `.ipynb` notebook used for the workspace template, refer to `examples/main.py`.\n",
+    "\n",
+    "**Note:** For a more general introduction to batch inference with Ray Data, check out the `Batch Inference Basics` workspace template.\n",
+    "\n",
+    "### How to decide between online vs offline inference for LLM\n",
+    "Online LLM inference (e.g. Anyscale Endpoint) should be used when you want to get real-time response for prompt. Use online inference when you want to optimize latency of inference to be as quick as possible.\n",
+    "\n",
+    "On the other hand, offline LLM inference should be used when you want to get reponses for a large number of prompts within an end-to-end time requirement (e.g. minutes to hours granularity). Use offline inference when you want to optimize throughput of inference to use resource (e.g. GPU) as much as possible on large-scale input data."
```

Comment:
it doesn't seem very necessary to mention "within an end-to-end time requirement". 
I think the most important factors are: 1) scalable to large data; 2) better throughput and GPU util.